src/axom/core/Array.hpp

```diff
@@ -340,6 +340,11 @@ class Array : public ArrayBase<T, DIM, Array<T, DIM, SPACE>>
 
   /// @}
 
+  /*!
+    \brief Returns spacing between adjacent items.
```

Comment:
Is "offset" a better term here? Is "spacing" a common term in this context? Is it in the units of data type size?

Reply:
I considered "offset" but decided against because it not the same as the offset we use in similar contexts, specifically the 1D offset and what RAJA calls offset.  I have not seen any other project name this quantity, so it's not a common term.  It's in the unit of data type size.

Reply:
OK. If others are fine with it, I'm OK with it also.  Is there a place in the documentation (source and user) where the basic concepts and terms for Array & ArrayView are defined? If not, that would be good to add to make clear what 'offset', 'size', 'spacing', etc. mean

Reply:
It seems like `stride` matches better with other contexts?

Reply:
And if its not really a `stride`, could it be ?

Reply:
Stride is also another good choice, if we didn't already use it for something else.  We pre-compute "strides" as the number of items to jump when i, j or k changes by 1.

Reply:
Yes - I am sorry, I keep forgetting this supports multiple interfaces.

Reply:
Documentation of "spacing" has been added to sphinx and the doxygen comments of `ArrayView`.  In practice, it's not really used by `Array`, always defaulting to 1 and should get optimized away by the compiler.  It's only in `Array` to let us optimize the 1D case of `ArrayBase`.